### ISSUE TITLE
Can't delete a comment

### ISSUE THREAD

**Issue by rk4an** [centreon/centreon#8306]
***Versions***

```
$ rpm -qa | grep centreon
centreon-plugin-Applications-Databases-Mysql-20191016-111959.el7.centos.noarch
centreon-plugin-Applications-Monitoring-Centreon-Central-20191016-111959.el7.centos.noarch
centreon-widget-tactical-overview-19.10.0-1.el7.centos.noarch
centreon-widget-graph-monitoring-19.10.0-1.el7.centos.noarch
centreon-license-manager-19.10.0-1.el7.centos.noarch
centreon-plugin-Applications-Databases-Mssql-20191016-111959.el7.centos.noarch
centreon-plugin-Operatingsystems-Windows-Snmp-20191016-111959.el7.centos.noarch
centreon-release-19.10-1.el7.centos.noarch
centreon-engine-extcommands-19.10.6-3.el7.centos.x86_64
centreon-widget-httploader-19.10.0-1.el7.centos.noarch
centreon-connector-ssh-19.10.0-1.el7.centos.x86_64
centreon-base-config-centreon-engine-19.10.1-3.el7.centos.noarch
centreon-plugin-Hardware-Printers-Generic-Snmp-20191016-111959.el7.centos.noarch
centreon-plugin-Applications-Monitoring-Centreon-Poller-20191016-111959.el7.centos.noarch
centreon-plugin-Applications-Protocol-Ftp-20191016-111959.el7.centos.noarch
centreon-engine-daemon-19.10.6-3.el7.centos.x86_64
centreon-widget-live-top10-memory-usage-19.10.0-1.el7.centos.noarch
centreon-connector-perl-19.10.0-1.el7.centos.x86_64
centreon-auto-discovery-server-19.10.0-2.el7.centos.noarch
centreon-plugin-Applications-Protocol-Dns-20191016-111959.el7.centos.noarch
centreon-plugin-Applications-Protocol-Ldap-20191016-111959.el7.centos.noarch
centreon-clib-19.10.0-6.el7.centos.x86_64
centreon-broker-cbmod-19.10.1-1.el7.centos.x86_64
centreon-widget-grid-map-19.10.0-1.el7.centos.noarch
centreon-widget-service-monitoring-19.10.1-1.el7.centos.noarch
centreon-widget-engine-status-19.10.0-1.el7.centos.noarch
centreon-trap-19.10.1-3.el7.centos.noarch
centreon-poller-centreon-engine-19.10.1-3.el7.centos.noarch
centreon-pp-manager-19.10.0-5.el7.centos.noarch
centreon-nrpe3-plugin-3.2.1-7.el7.centos.x86_64
centreon-plugin-Operatingsystems-Linux-Snmp-20191016-111959.el7.centos.noarch
centreon-plugin-Applications-Monitoring-Centreon-Database-20191016-111959.el7.centos.noarch
centreon-common-19.10.1-3.el7.centos.noarch
centreon-engine-19.10.6-3.el7.centos.x86_64
centreon-web-19.10.1-3.el7.centos.noarch
centreon-widget-live-top10-cpu-usage-19.10.0-1.el7.centos.noarch
centreon-connector-19.10.0-1.el7.centos.x86_64
centreon-19.10.1-3.el7.centos.noarch
centreon-plugin-Hardware-Ups-Standard-Rfc1628-Snmp-20191016-111959.el7.centos.noarch
centreon-broker-19.10.1-1.el7.centos.x86_64
centreon-broker-storage-19.10.1-1.el7.centos.x86_64
centreon-broker-cbd-19.10.1-1.el7.centos.x86_64
centreon-widget-host-monitoring-19.10.0-1.el7.centos.noarch
centreon-widget-servicegroup-monitoring-19.10.0-1.el7.centos.noarch
centreon-license-manager-common-19.10.0-1.el7.centos.noarch
centreon-plugin-Applications-Monitoring-Centreon-Map4-Jmx-20191016-111959.el7.centos.noarch
centreon-plugin-Network-Cisco-Standard-Snmp-20191016-111959.el7.centos.noarch
centreon-plugin-Applications-Protocol-Http-20191016-111959.el7.centos.noarch
centreon-perl-libs-19.10.1-3.el7.centos.noarch
centreon-broker-core-19.10.1-1.el7.centos.x86_64
centreon-widget-hostgroup-monitoring-19.10.0-1.el7.centos.noarch
centreon-plugin-Virtualization-VMWare-daemon-3.0.3-20190704131142.el7.centos.noarch
centreon-widget-global-health-19.10.0-1.el7.centos.noarch
centreon-database-19.10.1-3.el7.centos.noarch
```

***Operating System***

*CentOS*

***Browser used***

-  Google Chrome

Version:  79

### Description

Can't delete a comment

### Steps to Reproduce

1. I logged in Centreon
2. Go to downtime > comment
3. Delete a comment
4. Comment still here, event after waiting and refresing

No log...

**Comment by rk4an** [comment]
http://192.168.1.15/centreon/main.get.php?p=21002&num=7

![image](https://user-images.githubusercontent.com/322320/73774655-a7852f00-4784-11ea-81fd-b062f262c04f.png)

/usr/share/centreon/www/include/monitoring/comments/common-Func.php

![image](https://user-images.githubusercontent.com/322320/73774715-c71c5780-4784-11ea-9996-85ce30b9832c.png)

return Null

**Comment by sc979** [comment]
Thanks again @rk4an ,
The fix is available here : https://github.com/centreon/centreon/pull/8367,
and will be released in upcoming 18.10.13 / 19.04.12 / 19.10.9 versions
Best Regards
sc979
